src/style.css

```diff
@@ -0,0 +1,48 @@
+html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
`html` tag does not have any default margins or paddings
remove styles for `html`